spec/Knp/FriendlyContexts/Guesser/IntGuesserSpec.php

```diff
@@ -50,4 +57,26 @@ function it_should_transform_string_to_int()
         $this->transform("2.6")->shouldReturn(3);
         $this->transform(" ")->shouldReturn(0);
     }
+
+    function it_should_use_length_parameter_while_faking_an_int(Base $faker)
+    {
+        $mapping = [
+            'length' => 3,
+        ];
+
+        $faker->fake('numberBetween', [0, 999])->shouldBeCalled();
+
+        $this->fake($mapping);
+    }
+
+    function it_should_use_limit_of_2000000000_as_a_max_integer_value(Base $faker)
```

Comment:
is this name too specific? i know that the guy who wrote this originally was also a bit random using this value, but maybe it was just a quick choice because he couldn't be bothered to look up the exact size. but maybe baking into the name of a spec makes it seem like it has some "magic" meaning?